### PR TITLE
Bug fixes and openfisca-uk bump

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
### OpenFisca-UK bump

Increases the OpenFisca-UK version to `0.13.0`, containing the updated FRS weights.

### Bug fixes

Fixes a few UI-related bugs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 This repo consists of two packages - the React client and the Python server. A change to either repo should trigger an update in the versions for both to ensure a consistent changelog in this repo.
 
+# [1.19.0] - 2022-03-01
```
Comment (MaxGhenis) [inline CHANGELOG.md:10]: Seems like a patch

**Review by MaxGhenis — APPROVED** [review]
Only thought is that we might want to adjust the reweighting once more based on categories. Since this fixes other bugs, I'm OK merging it in and then merging a new reweighting in the next day or two as needed, up to you.

File: CHANGELOG.md (hunk 1 above)
Comment (nikhilwoodruff) [reply CHANGELOG.md]: Yeah, changed

**Comment by nikhilwoodruff** [comment]
Reducing the OpenFisca-UK version bump to `0.13.0`, which doesn't include the updated weights. Will merge this before the reweighting is finished.

**Pull request merged**